production/technical_userguide.md

```diff
@@ -666,3 +666,68 @@ curl https://api.bcda.cms.gov/data/45/99dbc86f-74e5-4553-88a9-af3e718cb72b.ndjso
 The response will be the requested data as [FHIR Coverage resources](https://www.hl7.org/fhir/coverage.html){:target="_blank"} in NDJSON format.
 
 An example of one such resource is available in the [guide to working with BCDA data](/data-guide/#sample-bcda-files).
+
+## Filtering Your Data with `_since`
+### About `_since` 
+`_since`, also known as "the since parameter," grants you the ability to apply a date parameter to your bulk data. Instead of receiving the full record of historical data every time you request data from an endpoint, you will be able to use `_since` to submit a date. BCDA will then produce data from the Patient, Coverage, or Explanation of Benefit endpoints that have been loaded since the entered date.
+
+For more information on ‘ _since’, please consult the [FHIR standard on query parameters](https://hl7.org/Fhir/uv/bulkdata/export/index.html#query-parameters).
+### Usage
+There are a couple of helpful points to keep in mind when using `_since`.
+#### Before Using `_since` 
+Before using `_since` for the first time, we recommend that you retrieve all historical data from the `/patient` and ‘/group’ endpoints. Once you have retrieved your historical data and begun using _since, you should use [_transactionTime_](https://hl7.org/Fhir/uv/bulkdata/export/index.html#response---complete-status) from your last `_since` call as the date for following `_since` calls.  This guarantees that there will be no gaps in the claims data you retrieve from BCDA. 
+
+**Note: Limitations of the Beneficiary FHIR Data (BFD) Server prevent data before 02-12-2020 from being tagged correctly, and  `_since` calls before that date may return gaps in data.**
```

Comment:
This is not true anymore.  BFD has tagged all historical data with an arbitrary date.